src/main/java/com/faforever/client/coop/CoopController.java

```diff
@@ -232,6 +236,12 @@ private void loadLeaderboard() {
         });
   }
 
+  private List<CoopResultBean> filterOnlyUniquePlayers(List<CoopResultBean> result) {
+    Set<String> uniquePlayerNames = new HashSet<>();
+    result.removeIf(coopResult -> !uniquePlayerNames.add(commaDelimitedPlayerList(coopResult)));
+    return result;
+  }
```

Comment:
This doesn't work if the players are in a different order. I would use something like an unmodifiable set as the key

Reply:
It won't work if we remove the sort rule:
https://github.com/FAForever/downlords-faf-client/blob/ccef5b08309528993f4ee2a4fdc6c778015d8413/src/main/java/com/faforever/client/coop/CoopService.java#L51

Reply:
That is sorting on the replay not on the players or the teams